Directory.Build.props

```diff
@@ -18,9 +18,4 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
Do you no longer need this for IISIntegration?

Reply:
It was already removed from dev, not sure why it is showing up as a diff.